tests/test_greedy_selector.py

```diff
@@ -118,23 +121,20 @@ def test_size_input(self):
         X = np.array([1, 2, 3, 4, 5]).reshape(-1, 1)
         selector_sample = GreedyTester(selection_type="sample")
         selector_feature = GreedyTester(selection_type="feature")
-        selector_sample.fit(X)
         with self.assertRaises(ValueError) as cm:
             selector_feature.fit(X)
-        self.assertEqual(
-            str(cm.exception),
-            "Found array with 1 feature(s) (shape=(5, 1)) while a minimum of 2 is "
-            "required.",
-        )
-        X = X.reshape(1, -1)
-        selector_feature.fit(X)
+            self.assertEqual(
```

Comment:
The check of the error message is on purpose outside of the assertRaises context. Otherwise you raise a ValueError get out of the context and skip the assert. See PR https://github.com/lab-cosmo/scikit-matter/pull/153

Reply:
fixed!